values.yaml

```diff
@@ -0,0 +1,44 @@
+# Default values for infinispan-helm-charts.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image: quay.io/infinispan/server:12.1.3
+initContainerImage: registry.access.redhat.com/ubi8-micro
+
+replicas: 1
+
+service:
+  type: DataGrid
```

Comment:
As the CacheService is going away in the operator I don't think that we should ever provide it in the helm-charts, so I would say we can remove this field and associated logic entirely and default to the DataGrid equivalent behaviour.

Similarly, maybe we should make the `container` and `resources` fields no longer children of "service". Instead we can combine these two and do something like:

```yaml
container:
  cpu: 500m
  memory: 512Mi
  storage: 1Gi
  storageClassName: ""
  ephemeralStorage: false
```

OR

```yaml
container:
  resources:
    cpu: 500m
    memory: 512Mi
  storage: 1Gi
  storageClassName: ""
  ephemeralStorage: false
```

I'm not sure where `extraJvmOpts` should belong though, maybe just a value at the root level?